cosmo/tests/test_netboxclient.py

```diff
@@ -16,46 +16,17 @@
     ]}
 
 
-def test_case_query_v3_ok(mocker):
-    utils.RequestResponseMock.patchTool(mocker)
-    nc = NetboxV3Strategy(TEST_URL, TEST_TOKEN)
-    nc.query('check')
-
-
-def test_case_query_v3_nok(mocker):
-    with pytest.raises(Exception):
-        utils.RequestResponseMock.patchTool(
-            mocker, graphqlData={'status_code': 403, 'text': 'unauthorized'})
-        nc = NetboxV3Strategy(TEST_URL, TEST_TOKEN)
-        nc.query('check')
-
-
-def test_case_query_v4_ok(mocker):
-    utils.RequestResponseMock.patchTool(mocker)
-    nc = NetboxV4Strategy(TEST_URL, TEST_TOKEN)
-    nc.query('check')
-
-
-def test_case_query_v4_nok(mocker):
-    with pytest.raises(Exception):
-        utils.RequestResponseMock.patchTool(
-            mocker, graphqlData={'status_code': 403, 'text': 'unauthorized'})
-        nc = NetboxV4Strategy(TEST_URL, TEST_TOKEN)
-        nc.query('check')
-
-
 def test_case_get_data(mocker):
-    mockAnswer = []
-    [versionDetectMock, dataMock] = utils.RequestResponseMock.patchTool(
-        mocker, graphqlData={'status_code': 200, 'text': '{"data":' + str(mockAnswer) + '}'})
+    mockAnswer = {
+        "device_list": [],
+        "l2vpn_list": [],
+        "vrf_list": [],
+        "loopbacks": {},
+    }
+    utils.RequestResponseMock.patchNetboxClient(mocker)
+
     nc = NetboxClient(TEST_URL, TEST_TOKEN)
+    assert nc.version == "4.1.2"
+
     responseData = nc.get_data(TEST_DEVICE_CFG)
     assert responseData == mockAnswer
-    versionDetectMock.assert_called_once()
-    dataMock.assert_called_once()
-    kwargs = dataMock.call_args.kwargs
-    assert 'json' in kwargs
-    assert 'query' in kwargs['json']
-    ncQueryStr = kwargs['json']['query']
-    for device in [*TEST_DEVICE_CFG['router'], *TEST_DEVICE_CFG['switch']]:
```

Comment:
same remark as in previous merge request, why did we delete this? I believe the assert responseData == mockAnswer is sufficient, right? (tip: I have no clue)